doc/new-tracing/tracers_doc_generated.md

```diff
@@ -4279,27 +4293,6 @@ Backends:
       `Forwarder`
 Filtered `Invisible` by config value: `Notice`
 
-### Forge.KESInfo
-
-
-> kesStartPeriod 
-> kesEndPeriod is kesStartPeriod + tpraosMaxKESEvo
-> kesEvolution is the current evolution or /relative period/.
-
-
-Severity:  `Info`
-Privacy:   `Public`
-Details:   `DNormal`
-
-
-From current configuration:
-
-Backends:
-      `EKGBackend`,
-      `Stdout MachineFormat`,
-      `Forwarder`
-Filtered `Visible` by config value: `Info`
-
```

Comment:
Some documentation bits are gone -- is that intended?

Reply:
That's true. I had to remove it, because a more principled approach lead to that the docu generator is not working for this special case. But I have an idea how to fix this, but will be part of next PR.